resources/phrase_lists/long_phrases.yml

```diff
@@ -0,0 +1,256 @@
+---
+type: phrases
+name: "parser-error"
+opts:
+- "Sorry, my parser is out of service."
+- "Oopsie. I could not represent your sentence in ones and zeros sequence. It must have been a long sentence."
+- "Oh wow, you a probably very intelligent. You sentence crashed my parser."
+- "Sorry, didnt quite get what you meant. Human language is tough and I am still learning."
+- "What a pity. I completely missed what you just said."
+- "Hm, seems like what you are saying is very smart. I have to grow up to understand it."
+- "Sorry, my parser seems to be out of service."
+- "I am sorry, I could not get what you are saying, it must be very loud here."
+- "Perhaps, I don't know a correct answer to this."
+- "I do not feel like answering this question, sorry."
+- "I am sorry I did not follow you."
+- "Sorry, I got so overwhelmed, I forgot the answer."
+---
+type: phrases
+name: "negative-sentiment"
+opts:
+- "let me tell you something different!"
+- "lets switch gears!"
+- "lets change the topic!"
+- "i have calculated the level of your engagement in the conversation. it is twenty five percent. Lets pivot now"
+- "i have something way better to talk about."
+---
+type: phrases
+name: "segue-avoid-answer"
+opts:
+- "the answer, well, only robots would understand."
+- "You know, I never thought about it."
+- "Access denied."
+- "Processing, wait for it, processing, oh well, forget about it"
+- "The answer is, uh, oh, Core dumped"
+- "The answer is 42 then."
+- "001."
+- "1001011."
```

Comment:
Question: If we get roboy to verbalize this, will he verbalize `1 million 1 thousand and 11` or `One Zero Zero ...`. I believe we want the later, but I believe the former would be verbalized

@josephbirkner @missxa